src/test/resources/collectionCreation.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name":"test-collection",
```

Comment:
I think maybe I don't need this bc I pass a collection name (`testCollection`) [here](https://github.com/pinecone-io/pinecone-java-client/pull/91/files#diff-a3ec968386997c2c981ec71a27c2f0cf920cdf9263e08eaa7485a8b6bdbce4c0R270)?

Reply:
Responded above, I think it's better to keep the name here and then use that in the test so we don't have two different representations of `name` for this fixture.